app/services/revocation_registry.py

```diff
@@ -309,61 +311,143 @@ async def revoke_credential(
             )
         )
     except ApiException as e:
-        bound_logger.info(
+        bound_logger.exception(
```

Comment:
Let's keep this an info log for now. Exception logs print a stacktrace, and for ACA-Py errors it's not helpful. It's info level because it could be 400 client errors. We'll inspect DataDog log frequency and update / breakdown logs further as needed